app/Resources/views/assistant/assistants.html.twig

```diff
@@ -7,12 +7,33 @@
 {% block body %}
     <div class="container-fluid assistants">
         <div class="content">
-            <header class="row page-header">
+            <header class="row page-header page-header-thin">
                 <div class="col-12">
                     {% include 'statictext/static_content.html.twig' with {'id': 'assistants-header'} %}
                 </div>
             </header>
         </div>
+
+        {% set departmentsWithActiveAdmission = [] %}
+        {% for department in departments %}
+            {% if department.currentSemester and department.currentSemester.hasActiveAdmission() %}
+                {% set departmentsWithActiveAdmission = departmentsWithActiveAdmission|merge([department.city]) %}
```

Comment:
Finnes det en repository function for å hente ut alle departments med active admission en plass?

Reply:
Nei, men det kan jeg fort lage. Men vi sender allerede inn et array med departments fra controlleren. Er det bedre å sende inn to arrayer, et med alle aktive og et bare med opptak, enn å ha denne business logikken?

Reply:
Ja, det er generelt sett bedre å flytte business logikk ut av viewet. Det har flere fordeler; viewet blir mer clean, logikken kan gjøres gjenbrukbar (f.eks. som en repository function), det blir enklere å skrive unit tester, og man følger prinsippet "separation of concerns" som anses som en best practice